templates/.net-desktop.yml

```diff
@@ -7,7 +7,7 @@ trigger:
 - master
 
 pool:
-  vmImage: 'VS2017-Win2016'
+  vmImage: 'Windows-latest'
```

Comment:
Aesthetically, I don't love the combination of capitalizing "Windows" but including the hyphen. It's a nit, though.